### PR TITLE
moved secrets to be part of cloudzero config, updated documentation to mathc

### PR DESCRIPTION
**Pull request by roberthocking**
By submitting a PR to this repository, you agree to the terms within the [CloudZero Code of Conduct](https://github.com/cloudzero/template-cloudzero-open-source/blob/master/CODE-OF-CONDUCT.md). Please see the [contributing guidelines](https://github.com/cloudzero/template-cloudzero-open-source/blob/master/CONTRIBUTING.md) for how to create and submit a high-quality PR for this repo.

### Description

> Describe the purpose of this PR along with any background information and the impacts of the proposed change. For the benefit of the community, please do not assume prior context.
>
> Provide details that support your chosen implementation, including: breaking changes, alternatives considered, changes to the API, etc.
>
> If the UI is being changed, please provide screenshots.

### References

> Include any links supporting this change such as a:
>
> - GitHub Issue/PR number addressed or fixed
> - StackOverflow post
> - Support forum thread
> - Related pull requests/issues from other repos
>
> If there are no references, simply delete this section.

### Testing

> Describe how this can be tested by reviewers. Be specific about anything not tested and reasons why. If this library has unit and/or integration testing, tests should be added for new functionality and existing tests should complete without errors.
>
> Please include any manual steps for testing end-to-end or functionality not covered by unit/integration tests.
>
> Also include details of the environment this PR was developed in (language/platform/browser version).

- [ ] This change adds test coverage for new/changed/fixed functionality

### Checklist

- [ ] I have added documentation for new/changed functionality in this PR
- [ ] All active GitHub checks for tests, formatting, and security are passing
- [ ] The correct base branch is being used, if not `main`

### CODE REVIEW

File: charts/cloudzero-prometheus-agent/README.md
Comment (dmepham) [inline charts/cloudzero-prometheus-agent/README.md:32]: the chart name is just `cloudzero-agent` now. sorry, I think I forgot to update that when I changed this

File: charts/cloudzero-prometheus-agent/README.md
Comment (dmepham) [inline charts/cloudzero-prometheus-agent/README.md:85]: I think this is still using the `credentials.` bit in the description here

File: charts/cloudzero-prometheus-agent/templates/_helpers.tpl
Comment (dmepham) [inline charts/cloudzero-prometheus-agent/templates/_helpers.tpl:23]: isn't this in the `cloudzero` block now?

File: charts/cloudzero-prometheus-agent/templates/_helpers.tpl
Comment (dmepham) [inline charts/cloudzero-prometheus-agent/templates/_helpers.tpl:17]: sorry, I think I added this named template but didn't use it. we should either get rid of it or use it in the configmap

File: charts/cloudzero-prometheus-agent/templates/cm.yaml
Comment (dmepham) [inline charts/cloudzero-prometheus-agent/templates/cm.yaml:139]: are `clusterName` and `cloudAccountId` for the params correct? I think we'd want to keep those. we didn't change that in the query params we set in our API

File: charts/cloudzero-prometheus-agent/templates/secret.yaml
Comment (dmepham) [inline charts/cloudzero-prometheus-agent/templates/secret.yaml:2]: I think this goes in the `cloudzero` block

File: charts/cloudzero-prometheus-agent/templates/secret.yaml
Comment (dmepham) [inline charts/cloudzero-prometheus-agent/templates/secret.yaml:14]: same here - are we putting this in the `cloudzero` block? maybe I just don't know what we decided on

**Review by dmepham — CHANGES_REQUESTED** [review]
I think some of my comments don't apply, as I'm not sure what the final decision was on where `apiKey` should live

File: charts/cloudzero-prometheus-agent/README.md
Comment (roberthocking) [reply charts/cloudzero-prometheus-agent/README.md]: Fixed

File: charts/cloudzero-prometheus-agent/values.yaml
Comment (roberthocking) [reply charts/cloudzero-prometheus-agent/values.yaml:9]: Moved them to top level settings. We can revisit this based on additional feedback, but it's good to go now.

File: charts/cloudzero-prometheus-agent/templates/secret.yaml
Comment (roberthocking) [reply charts/cloudzero-prometheus-agent/templates/secret.yaml:14]: We removed the cloudzero block

File: charts/cloudzero-prometheus-agent/templates/secret.yaml
Comment (roberthocking) [reply charts/cloudzero-prometheus-agent/templates/secret.yaml:2]: We removed the cloudzero block

File: charts/cloudzero-prometheus-agent/templates/cm.yaml
Comment (roberthocking) [reply charts/cloudzero-prometheus-agent/templates/cm.yaml:139]: Ooops.

File: charts/cloudzero-prometheus-agent/values.yaml
Comment (dmepham) [reply charts/cloudzero-prometheus-agent/values.yaml:9]: great, yeah I like that format

File: charts/cloudzero-prometheus-agent/README.md
Comment (roberthocking) [reply charts/cloudzero-prometheus-agent/README.md]: removed all references to the credentials in the description

File: charts/cloudzero-prometheus-agent/templates/_helpers.tpl
Comment (roberthocking) [reply charts/cloudzero-prometheus-agent/templates/_helpers.tpl:17]: let's remove it.

**Review by dmepham — APPROVED** [review]
lgtm, just some minor things. we can also merge this and fix in a follow up PR if you'd prefer that

File: charts/cloudzero-agent/README.md
Comment (roberthocking) [reply charts/cloudzero-agent/README.md:33]: fixed

File: charts/cloudzero-agent/README.md
Comment (roberthocking) [reply charts/cloudzero-agent/README.md:48]: fixed

File: charts/cloudzero-agent/values.schema.json
Comment (roberthocking) [reply charts/cloudzero-agent/values.schema.json:36]: Makes sense. I'll remove it and we can gather feedback and see what works best here.

**Pull request merged**
